touch: update page (#12789)
tldr-pages/tldr: pages/common/touch.md

`touch -c -t {{YYYYMMDDHHMM.SS}} {{path/to/file1 path/to/file2 ...}}`

> - Set the files' timestamp to the [r]ef_file's timestamp, and don't [c]reate file if not exist:

Reviewer: ```suggestion
- Set the files' timestamp to the [r]eference file's timestamp, and do not [c]reate the file if it does not exist:
```

I assume that ref_file stands for reference file?
